pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -198,6 +233,13 @@ func TestConnectionStore_addAndUpdateConn(t *testing.T) {
 			expConn.DestinationPodNamespace = "ns2"
 			expConn.DestinationPodName = "pod2"
 		case 2:
+			// Tests update part of the function for dying connection.
+
+			expConn.SourcePodNamespace = "ns3"
+			expConn.SourcePodName = "pod3"
+			expConn.TCPState = "TIME_WAIT"
+			expConn.StopTime = refTime.Add(-(time.Second * 30))
+		case 3:
```

Comment:
no need to address it in this PR, but I think this is a poor way to write tests. If the tests are that different, then it just means that the table pattern is not adapted, and you should write separate tests / separate sub-tests